lib/game.rb

```diff
@@ -0,0 +1,27 @@
+class Game
```

Comment:
Good use of SRP, by creating a game class to store all your ruby functionality inside.